lib/ansible/constants.py

```diff
@@ -265,7 +265,7 @@ def load_config_file():
 # Become
 BECOME_ERROR_STRINGS      = {'sudo': 'Sorry, try again.', 'su': 'Authentication failure', 'pbrun': '', 'pfexec': '', 'doas': 'Permission denied', 'dzdo': '', 'ksu': 'Password incorrect'} #FIXME: deal with i18n
 BECOME_MISSING_STRINGS    = {'sudo': 'sorry, a password is required to run sudo', 'su': '', 'pbrun': '', 'pfexec': '', 'doas': 'Authorization required', 'dzdo': '', 'ksu': 'No password given'} #FIXME: deal with i18n
-BECOME_METHODS            = ['sudo','su','pbrun','pfexec','doas','dzdo','ksu']
+BECOME_METHODS            = ['sudo','su','pbrun','pfexec','doas','dzdo','ksu','runas']
```

Comment:
Every BECOME_METHOD except runas has a BECOME_ERROR_STRINGS and BECOME_MISSING_STRINGS.  Is it expected to exist for 'runas', too?

Reply:
Yes, runas is different from most of the other become methods. If you are having an issue with it, please raise a GH issue with the details.